services/TotemAccountGateway.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.Events;
+using System.Net;
+using utilities;
+
+
+namespace TotemServices
+{
+    public class TotemAccountGateway : MonoBehaviour
+    {
+        private readonly string _urlRoot = "https://account.totem.gdn/";
```

Comment:
Do you thing those should be in an .env or similar config file instead?

Reply:
Yes, thats a good idea to keep this string-links in an env const file.